src/main/java/dao/CustomerDAOImpl.java

```diff
@@ -0,0 +1,97 @@
+package dao;
+
+import entities.Customer;
+import services.ConnectionFactory;
+
+import java.sql.*;
+import java.util.HashSet;
+import java.util.Set;
+
+public class CustomerDAOImpl implements DAO<Customer>{
+
+    private Customer extractCustomerFromResultSet(ResultSet rs) throws SQLException {
+        Customer customer = new Customer();
+        customer.setCustomerId(rs.getInt("idCustomer"));
+        customer.setAdmin(rs.getBoolean("isAdmin"));
```

Comment:
customer.isAdmin? sounds fun. we don't have admin role so you can just remove it

Reply:
Okey, I will remove this one